docs/service-brokers/docs/011-configuration-helm-broker.md

```diff
@@ -33,12 +33,33 @@ Follow these steps to change the configuration and make the Helm Broker fetch bu
 
     - A `{bundle_name}-{bundle_version}.tgz` file for each bundle version defined in the `index.yaml` file. The `.tgz` file is an archive of your bundle's directory.
 
-2. In the `values.yaml` provide your server's URL in the **config.repository.baseURL** attribute:
+2. In the `values.yaml` provide your server's URL in the **repository.baseURL** attribute:
 
   ```yaml
-  config:
     repository:
       baseURL: "http://custom.bundles-repository"
   ```
 
 3. Install Kyma on Minikube. See the **Local Kyma installation** document to learn how.
+
+### Configuring repository URL in the runtime
+
+Follow these steps to change the repository URL:
+
+1. Configure Helm Broker:
```

Comment:
`APP_REPOSITORY_BASE_URL="http://custom.bundles-repository/"` -> missing " at the end?